source/HitboxAnimationManager.hx

```diff
@@ -0,0 +1,125 @@
+package;
```

Comment:
This whole class may be more trouble than it's worth since it doesn't appear to be working as intended.  Currently, the hitbox for the sword does not rotate along with the graphic.
![image](https://cloud.githubusercontent.com/assets/14339030/26087510/c315c448-39b6-11e7-81db-a3439a35c341.png)
It might be better (simpler, at least, and reasonably close to the intended functionality) to simply resize the hitbox during the swing animation to a rectangle that covers the whole area of the swing.